nightly-examples/examples/echo.rs

```diff
@@ -0,0 +1,89 @@
+//! A "hello world" echo server with Tokio
```

Comment:
It would be good if the docs here had a link to the original example from `tokio`.